go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/ONSdigital/log.go v1.0.1
 	github.com/golang/mock v1.4.4
 	github.com/gorilla/mux v1.8.0
+	github.com/mattn/go-isatty v0.0.12 // indirect
```

Comment:
I think this can be removed - `go mod tidy`?

Reply:
had to remove it manually, then go mod tidy removed it from go.sum